Makefile

```diff
@@ -238,7 +239,11 @@ rss: $(VENV) doc chsdi/static/doc/build/releasenotes/index.html
 
 .PHONY: legends
 legends: $(VENV) guard-BODID guard-WMSHOST
-	scripts/downloadlegends.sh $(WMSHOST) $(BODID) $(WMSSCALELEGEND)
+	WMSPROTOCOL="https"; \
```

Comment:
or we take the whole url into WMSHOST, including protocol.
`WMSHOST=https://localhost:9X78`

Reply:
I did not want to get into .env.default vars ...